lib/hooks/useUsersAddress.js

```diff
@@ -1,17 +1,8 @@
-import { atom, useAtom } from 'jotai'
 import { WalletContext } from 'lib/components/WalletContextProvider'
-import { useContext, useEffect } from 'react'
-
-export const usersAddressAtom = atom('')
+import { useContext } from 'react'
 
+// TODO: Add ENS!!
```

Comment:
What part of ENS do we want here? It sounds important!! 😄

Reply:
Nothing super actionable at the moment but:

We should show it here like flagship. 
<img width="134" alt="Screen Shot 2021-03-29 at 4 00 09 PM" src="https://user-images.githubusercontent.com/13701258/112892870-e1f80c80-90a7-11eb-8e78-df2ebca763a9.png">

But it'd also be nice to have everywhere else we display a users address. Owner of a pool, past winners (if/when that gets in the ref app)